docs/admin-guide/migration/in-place/index.md

```diff
@@ -0,0 +1,186 @@
+when you install the latest dependencies.
+
+* [ ] MySQL 5.7
+    * [ ] /etc/mysql/mysql.conf.d/mysql.cnf
+    * [ ] /etc/mysql/mysql.conf.d/mysqld.cnf
+* [ ] Tomcat 7
+    * [ ] /opt/tomcat/bin/setenv.sh 
+    * [ ] /opt/tomcat/conf/server.xml
+    * [ ] /etc/systemd/system/tomcat.service
+* [ ] Java 7
+* [ ] Apache 2.2
+    *  [ ] /etc/apache2/sites-enabled/000-default-le-ssl.conf
+* [ ] OpenBoxes v0.8.x 
+    * [ ] /opt/tomcat/.grails/openboxes-config.properties
+    * [ ] /opt/tomcat/.grails.openboxes-config.groovy
+
+!!! tip 
+    One easy way to remind yourself what you've changed on a particular server would be to go 
+    through your Bash History and see what files have been changed. 
+
+    The following command will show you all of the files you've opened with vi, which could lead
+    to some 
+
+        $ history | grep vi
+
+## Perform Migration
+
+
+### Java 
+
+=== "Zulu"
+
+        sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 0xB1998361219BD9C9
+        sudo apt-add-repository 'deb http://repos.azulsystems.com/ubuntu stable main' -y
+        sudo apt-get update
+        sudo apt-get install zulu-8 -y
+
+
+### Application Server
+
+!!! todo
+
+    I would prefer if we install Tomcat under /opt and then create a symbolic link /opt/tomcat to 
+    the version you want to use. 
+
+=== "Tomcat 9 (APT)"
+
+        sudo apt install tomcat9 
+
+=== "Tomcat 9 (Manual)"
+
+        wget https://archive.apache.org/dist/tomcat/tomcat-9/v9.0.91/bin/apache-tomcat-9.0.91.tar.gz
+        tar xfv apache-tomcat-9.0.91.tar.gz
+        sudo mv apache-tomcat-9.0.91 /opt/tomcat9
+        sudo chown -R tomcat:tomcat /opt/tomcat9
+        sudo chmod +x /opt/tomcat9/bin/*
+        rm apache-tomcat-9.0.91.tar.gz
+
+=== "Tomcat 8.5 (Manual)"
+
+        wget https://archive.apache.org/dist/tomcat/tomcat-8/v8.5.89/bin/apache-tomcat-8.5.89.tar.gz
+        tar xfv apache-tomcat-8.5.89.tar.gz
+        sudo mv apache-tomcat-8.5.89 /opt/tomcat85
+        sudo chown -R tomcat:tomcat /opt/tomcat85
+        sudo chmod +x /opt/tomcat85/bin/*
+        rm apache-tomcat-8.5.89.tar.gz
+
+### Web Server / HTTPS / SSL (optional)
+This step is optional since you can stay with Apache 2, if that's what was installed previously. We
+have started to migrate some of our production environments to use nginx, so that's why these
+instructions are included.
+
+=== "Apache"
+
+    If you've configured SSL/HTTPS on Apache 2 then you can stay with that configuration. You might 
+    need to re-configure Apache 2 to delegate requests to Tomcat 8.5 or Tomcat 9 through mod_jk or 
+    whatever plugin was used. We'll try to upgrade the docs to include those instructions soon.
+    
+    !!! todo
+    
+            Include configuration changes for mod_jk or mod_http or whatever. 
+
+=== "nginx"
+    
+    #### Install nginx apt
+    
+        sudo apt-get install nginx -y
+    
+    #### Configure SSL
```

Comment:
if you can figure out how to work that in, then great. if you'd prefer to remove the "in place" migration instructions because there are a lot of gotchas, then i'd be ok with that. and i've been thinking similarly since starting down this path.

Reply:
@jmiranda I figured out that the missing part was to install the proper certbot plugin for nginx:
```
sudo apt-get install python3-certbot-nginx
```